docs/source/examples/train_loop.rst

```diff
@@ -149,6 +149,10 @@ The main steps are twofold:
                 return self.l3(h)
 
         model = MyNetwork()
+        
+        gpu_id = -1 # Set to -1 if you use CPU, or >= 0 to use gpu.
```

Comment:
* Please use `gpu_id = 0`
* Please insert 2-spaces between code and comment (`#`)